File: MVVM/WindowViewModel.vb
Imports Ks.Common.Controls

Namespace MVVM

    <ViewModelMetadata(GetType(Window), IsSingleInstance:=True)>
    Public Class WindowViewModel
        Inherits NavigationViewModel

        Public Sub New(ByVal KsApplication As KsApplication)
            MyBase.New(KsApplication)
        End Sub

    End Class

End Namespace

File: UI/WPF/MVVM/WindowViewModel.cs
using System.Windows;

using Ks.Common.Controls;

namespace Ks.Common.MVVM
{
    [ViewModelMetadata(typeof(Controls.Window), IsSingleInstance = true)]
    public class WindowViewModel : NavigationViewModel
    {
        public WindowViewModel(KsApplication KsApplication) : base(KsApplication)
        {
        }

        public WindowViewModel() : base()
        {
        }
    }
}
